### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**
- [DEMO LINK](https://maksym-nemera.github.io/layout_stars/)
- [TEST REPORT LINK](https://maksym-nemera.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,7 +7,56 @@
...
   <body>
-    <h1>Stars</h1>
+    <div class="all-stars">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
```
Comment (naziamoff) [inline src/index.html:19]: would be great to use semantic tags here. You got a list with list items here, right?

File: src/index.html [hunk 2]
```diff
@@ -7,7 +7,56 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Stars</h1>
+    <div class="all-stars">
```
Comment (naziamoff) [inline src/index.html:12]: image

If that's not what you mean - it is better to rename this class for sth like stars__container

**Review by naziamoff — APPROVED** [review]
Awesome 👍 

See some minor suggestions below

File: src/index.html (hunk 1 above)
Comment (NickMitchell61) [reply src/index.html:19]: You've recommended change <div> on ul>li*5, Am I clearly understanding you?

File: src/index.html (hunk 2 above)
Comment (NickMitchell61) [reply src/index.html:12]: Got it!

**Pull request closed without merging**